BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -41,6 +46,15 @@
   private List<NoteItem> noteList;
   private List<NoteItem> selectedNoteItems;
 
+  private BookDao bookDao;
+  private NoteDao noteDao;
+
+  private ExportBibTex exportBibTex;
+  private String fileName;
+  private final String folderName = "Download";
+  private final String fileTypeBibToast = ".bib";
+
+  private static final int STORAGE_PERMISSION_CODE = 1;
```

Comment:
The folderName, fileTypeBibToast and STORAGE_PERMISSION_CODE always have the same values. Maybe you can create a ExportConstants class for them (e.g. see LibraryKeys class) and also use them in your other export code. Then you could use your constants like this: ExportConstants.STORAGE_PERMISSION_CODE